src/components/SelectSearch.tsx

```diff
@@ -45,58 +41,63 @@ export default class SelectSearch extends React.PureComponent<
         super(props);
 
         this.state = {
-            selection: props.initial
+            selections: props.initial
```

Comment:
i think `selections` is more appropriate here because `onChange` will be passed either a single value or an array of values.